synapse/config/saml2_config.py

```diff
@@ -256,6 +256,9 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
             #  remote:
             #    - url: https://our_idp/metadata.xml
 
+            # Allowed clock difference in seconds from IdP.
+            #accepted_time_diff: 3
```

Comment:
It would be nice to include what the default is here and happens when this is uncommented.

I'm of two minds about whether we want to include these options here and just point to the pysaml2 docs somewhere though. Including them is certainly a bit simpler for our users though.

Reply:
@clokep There is no default value here. The value of 3 seconds has proved to be useful for me in other SPs. It depends on how big the time difference between SP and IdP is or can be. I have submitted a PR for this option, because errors caused by a time difference are sometimes hard to understand.

Yes, instead of taking every option from pysaml2, you could simply refer to the documentation. But as with the ui_info PR [#8718](https://github.com/matrix-org/synapse/pull/8718), there are options that are hardly documented at all. Therefore I think it is helpful if these options are given as an example to minimize a long search for solutions. Time differences between SP and IdP are a particularly nasty problem, because they may occur only sporadically and are not always replicable by the admin. To find out how mdui elements work in the pysaml2 configuration took me several hours because it is not documented at all.

Reply:
@clokep  Patrick, do I still have to do or add anything here? :)

Reply:
> @clokep There is no default value here. The value of 3 seconds has proved to be useful for me in other SPs. It depends on how big the time difference between SP and IdP is or can be. I have submitted a PR for this option, because errors caused by a time difference are sometimes hard to understand.

That means the default is 0 seconds, I think! 😄 

I think adding an additional line:

> Uncomment the below to increase the accepted time difference from 0 to 3 seconds.

Reply:
@clokep added your additional line.